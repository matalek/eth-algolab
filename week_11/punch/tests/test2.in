20
100 78
7615 9
7615 9
7615 9
6404 8
6404 8
6404 8
3545 5
3545 5
3545 5
3545 5
8814 4
8814 4
8814 4
8814 4
8814 4
8814 4
8814 4
8814 4
4439 3
4439 3
4439 3
4439 3
4439 3
4439 3
4439 3
4439 3
4439 3
4439 3
6819 5
6819 5
6819 5
6819 5
5733 3
5733 3
5733 3
5733 3
5733 3
5733 3
5733 3
5733 3
5733 3
1024 1
8207 6
8207 6
8207 6
8207 6
8207 6
3878 8
3878 8
3878 8
3878 8
4011 9
4011 9
2914 3
2914 3
2914 3
2914 3
2914 3
2914 3
8144 10
8144 10
8144 10
8144 10
8144 10
8144 10
8144 10
8144 10
8144 10
8144 10
723 8
723 8
723 8
723 8
723 8
723 8
723 8
723 8
6814 3
6814 3
6814 3
6814 3
6814 3
6814 3
6814 3
539 8
539 8
539 8
539 8
539 8
539 8
943 10
3645 10
4022 6
5458 3
9617 3
2804 7
4680 7
5857 1
6808 8
4733 1
100 81
1153 3
1153 3
8059 7
8059 7
8059 7
8059 7
8059 7
9659 10
9659 10
9659 10
9659 10
9659 10
9659 10
9659 10
9659 10
258 2
258 2
258 2
258 2
258 2
258 2
258 2
6646 6
6646 6
6646 6
6646 6
6646 6
6646 6
6646 6
6646 6
6817 1
6817 1
5422 9
5422 9
8146 1
8146 1
8146 1
8146 1
8146 1
4525 9
4525 9
4727 7
4727 7
6278 4
7763 4
7763 4
7763 4
7763 4
7763 4
7763 4
5721 7
3628 8
3628 8
3628 8
3628 8
4614 9
4614 9
4614 9
4614 9
4484 2
4484 2
4484 2
4484 2
4484 2
4484 2
7064 4
7064 4
7064 4
7064 4
7064 4
7064 4
7064 4
7064 4
3419 1
3419 1
3419 1
3419 1
3419 1
3419 1
3419 1
3419 1
3419 1
7199 7
7199 7
8642 6
8642 6
8642 6
3823 1
3823 1
6445 3
238 1
7299 4
9955 8
4331 10
4079 9
7733 5
1681 1
1966 8
9947 10
9177 9
100 92
5568 73
5568 73
5568 73
5568 73
5568 73
5568 73
9562 74
9562 74
9562 74
9562 74
9562 74
9562 74
7461 61
7461 61
7461 61
7461 61
7461 61
7461 61
8018 91
7543 75
7543 75
5875 56
5875 56
5875 56
5875 56
5875 56
5875 56
9858 99
9858 99
1932 84
1932 84
1932 84
1932 84
6402 52
6402 52
6402 52
6402 52
6402 52
6402 52
6444 83
6444 83
6444 83
4470 94
4470 94
4470 94
4470 94
4470 94
4470 94
4470 94
4470 94
6357 64
6357 64
6357 64
6357 64
6357 64
6357 64
6357 64
6357 64
6357 64
6357 64
560 100
560 100
560 100
560 100
560 100
560 100
560 100
560 100
560 100
560 100
3363 94
3363 94
3363 94
3363 94
3363 94
3363 94
3286 75
3286 75
3286 75
3286 75
3134 99
3134 99
1919 98
1919 98
1919 98
1919 98
1919 98
5370 56
5370 56
5370 56
5370 56
5370 56
5370 56
5370 56
5370 56
5370 56
5370 56
3993 58
2811 98
2096 100
100 79
5436 80
5436 80
5436 80
5436 80
5436 80
5436 80
5436 80
3088 63
3088 63
3088 63
1006 90
1006 90
1006 90
1006 90
1006 90
2916 74
2916 74
2916 74
2622 56
2622 56
2622 56
2000 87
2000 87
2000 87
2000 87
2000 87
2000 87
6229 86
6229 86
6229 86
6229 86
6545 54
6545 54
6545 54
6545 54
6545 54
6545 54
6545 54
4822 85
6682 78
1159 87
1159 87
5236 61
5236 61
4880 73
4880 73
4880 73
912 61
912 61
912 61
8021 83
8021 83
8021 83
8021 83
1294 60
2901 75
2901 75
2901 75
2901 75
2901 75
2901 75
2901 75
2912 76
2912 76
3694 60
3694 60
3694 60
3694 60
3694 60
8629 63
8629 63
8629 63
8629 63
8629 63
8629 63
8629 63
8629 63
3708 52
3708 52
3708 52
7679 79
7679 79
7679 79
7679 79
7679 79
4948 77
4948 77
4948 77
4535 52
4535 52
4535 52
4535 52
4535 52
4535 52
4535 52
4535 52
4535 52
1333 79
5805 95
213 91
100 55
6765 89
6765 89
6765 89
6765 89
6765 89
6765 89
6765 89
6765 89
9222 95
7622 87
7622 87
7622 87
6891 60
6891 60
6891 60
7785 50
7785 50
7785 50
7785 50
7785 50
7785 50
745 59
745 59
745 59
745 59
1218 64
1218 64
7833 74
7833 74
7833 74
7833 74
7833 74
7833 74
7833 74
7833 74
7833 74
7833 74
7203 69
9221 99
9221 99
6440 79
6440 79
1428 99
1428 99
1428 99
1428 99
1428 99
1428 99
3747 53
3747 53
8287 52
8287 52
8287 52
8287 52
8287 52
8287 52
7916 71
7916 71
7916 71
7916 71
7916 71
7916 71
7916 71
7916 71
7916 71
7916 71
1768 96
1768 96
1768 96
1768 96
1768 96
1768 96
1768 96
1768 96
1768 96
8887 62
6347 92
6347 92
6347 92
6347 92
6347 92
6347 92
6347 92
6347 92
6347 92
6347 92
8708 71
8708 71
8708 71
8708 71
8708 71
8708 71
8708 71
8708 71
8708 71
8708 71
2675 79
7339 67
1811 53
6379 62
100 88
3367 53
3367 53
3367 53
3367 53
3367 53
3367 53
7682 88
1153 35
1153 35
1153 35
1153 35
1153 35
1153 35
7279 42
7279 42
7279 42
7060 88
7060 88
7060 88
7928 39
7928 39
7928 39
7928 39
7928 39
7928 39
7928 39
7928 39
7108 1
7108 1
7108 1
237 71
237 71
237 71
237 71
1487 56
1487 56
1487 56
1487 56
1487 56
3659 34
3659 34
3659 34
3659 34
8811 67
8811 67
8811 67
8811 67
7074 80
7074 80
7074 80
7074 80
7074 80
7074 80
7074 80
7074 80
9 40
9 40
9 40
9 40
5373 48
7446 48
5465 48
7664 48
19 15
15 15
16 15
16 15
16 73
19 73
14 73
11 73
2409 13
7509 2
5462 82
8184 93
9263 24
8230 4
8585 6
9593 21
688 70
593 41
1777 54
2949 69
1439 99
2676 97
8494 45
6082 50
4284 13
1111 92
1224 42
8792 25
4757 9
9055 31
2707 86
3795 53
9215 49
5686 100
6226 81
9887 83
4095 63
100 68
9945 47
9945 47
9945 47
9945 47
9945 47
9945 47
1078 44
1078 44
1078 44
1078 44
3077 93
3077 93
3077 93
3077 93
3077 93
3077 93
3077 93
3077 93
3077 93
7634 13
7634 13
7634 13
7634 13
7634 13
2918 90
2918 90
2918 90
2918 90
2918 90
2918 90
7083 20
7083 20
7083 20
7083 20
7083 20
7083 20
1814 59
1814 59
1814 59
1814 59
5787 50
5787 50
2531 5
2531 5
2531 5
2531 5
2531 5
2531 5
2531 5
2531 5
2531 5
2531 5
1 8
1 8
1 8
1 8
9543 60
5038 60
6413 60
7565 60
16 10
11 10
14 10
18 10
18 58
16 58
12 58
13 58
2760 82
4349 21
8077 16
2623 91
8368 5
4834 52
1926 43
3086 97
3463 59
8318 93
7310 38
8623 67
3529 90
1892 29
7194 49
1432 1
5867 18
6724 46
1604 13
6344 54
4546 78
9089 49
6393 73
5345 65
6927 100
5531 6
6406 11
6136 48
4930 22
9137 66
5688 3
8935 88
100 87
1 15
1 15
1 15
1 15
1 15
1 15
1 15
1 15
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 11
1 11
1 11
1 11
1 11
1 11
1 11
1 11
1 13
1 13
1 13
1 13
1 13
1 17
1 17
1 17
1 17
1 17
1 17
1 8
1 8
1 8
1 8
1 8
1 8
1 8
1 8
1 5
1 5
1 5
1 5
1 5
1 5
1 5
1 13
1 13
1 13
1 18
1 18
1 18
1 18
1 18
1 18
1 18
1 11
1 11
1 11
1 11
1 11
1 11
1 11
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 18
1 18
1 18
1 18
1 18
1 18
1 18
1 11
1 2
1 16
1 12
1 5
1 8
1 16
100 52
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 18
1 18
1 15
1 15
1 15
1 15
1 15
1 10
1 10
1 10
1 10
1 10
1 10
1 10
1 17
1 17
1 17
1 17
1 17
1 17
1 17
1 6
1 5
1 5
1 5
1 5
1 5
1 5
1 5
1 5
1 5
1 12
1 13
1 13
1 13
1 13
1 3
1 3
1 3
1 3
1 3
1 3
1 3
1 16
1 20
1 20
1 20
1 20
1 20
1 20
1 20
1 20
1 5
1 5
1 5
1 5
1 5
1 5
1 5
1 17
1 9
1 9
1 9
1 9
1 18
1 7
1 7
1 7
1 7
1 7
1 7
1 7
1 7
1 7
1 7
1 7
1 7
1 7
1 5
1 17
1 17
1 17
1 17
1 17
1 17
1 17
1 19
1 14
1 12
1 12
100 69
1 12
1 12
1 12
1 12
1 12
1 12
1 2
1 2
1 2
1 2
1 2
1 2
1 9
1 9
1 9
1 9
1 9
1 9
1 9
1 9
1 9
1 2
1 2
1 2
1 2
1 2
1 2
1 2
1 11
1 11
1 11
1 11
1 11
1 11
1 18
1 18
1 18
1 18
1 18
1 18
1 18
1 10
1 10
1 10
1 10
1 10
1 10
1 10
1 10
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 5
1 5
1 6
1 6
1 6
1 6
1 6
1 6
1 6
1 6
1 6
1 20
1 9
1 9
1 9
1 9
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 18
1 18
1 18
1 18
1 18
1 18
1 18
1 18
1 18
1 18
1 1
100 7797
7615 9
7615 9
7615 9
6404 8
6404 8
6404 8
3545 5
3545 5
3545 5
3545 5
8814 4
8814 4
8814 4
8814 4
8814 4
8814 4
8814 4
8814 4
4439 3
4439 3
4439 3
4439 3
4439 3
4439 3
4439 3
4439 3
4439 3
4439 3
6819 5
6819 5
6819 5
6819 5
5733 3
5733 3
5733 3
5733 3
5733 3
5733 3
5733 3
5733 3
5733 3
1024 1
8207 6
8207 6
8207 6
8207 6
8207 6
3878 8
3878 8
3878 8
3878 8
4011 9
4011 9
2914 3
2914 3
2914 3
2914 3
2914 3
2914 3
8144 10
8144 10
8144 10
8144 10
8144 10
8144 10
8144 10
8144 10
8144 10
8144 10
723 8
723 8
723 8
723 8
723 8
723 8
723 8
723 8
6814 3
6814 3
6814 3
6814 3
6814 3
6814 3
6814 3
539 8
539 8
539 8
539 8
539 8
539 8
943 10
3645 10
4022 6
5458 3
9617 3
2804 7
4680 7
5857 1
6808 8
4733 1
100 8118
1153 3
1153 3
8059 7
8059 7
8059 7
8059 7
8059 7
9659 10
9659 10
9659 10
9659 10
9659 10
9659 10
9659 10
9659 10
258 2
258 2
258 2
258 2
258 2
258 2
258 2
6646 6
6646 6
6646 6
6646 6
6646 6
6646 6
6646 6
6646 6
6817 1
6817 1
5422 9
5422 9
8146 1
8146 1
8146 1
8146 1
8146 1
4525 9
4525 9
4727 7
4727 7
6278 4
7763 4
7763 4
7763 4
7763 4
7763 4
7763 4
5721 7
3628 8
3628 8
3628 8
3628 8
4614 9
4614 9
4614 9
4614 9
4484 2
4484 2
4484 2
4484 2
4484 2
4484 2
7064 4
7064 4
7064 4
7064 4
7064 4
7064 4
7064 4
7064 4
3419 1
3419 1
3419 1
3419 1
3419 1
3419 1
3419 1
3419 1
3419 1
7199 7
7199 7
8642 6
8642 6
8642 6
3823 1
3823 1
6445 3
238 1
7299 4
9955 8
4331 10
4079 9
7733 5
1681 1
1966 8
9947 10
9177 9
100 9197
5568 7311
5568 7311
5568 7311
5568 7311
5568 7311
5568 7311
9562 7384
9562 7384
9562 7384
9562 7384
9562 7384
9562 7384
7461 6116
7461 6116
7461 6116
7461 6116
7461 6116
7461 6116
8018 9040
7543 7468
7543 7468
5875 5679
5875 5679
5875 5679
5875 5679
5875 5679
5875 5679
9858 9854
9858 9854
1932 8384
1932 8384
1932 8384
1932 8384
6402 5240
6402 5240
6402 5240
6402 5240
6402 5240
6402 5240
6444 8238
6444 8238
6444 8238
4470 9376
4470 9376
4470 9376
4470 9376
4470 9376
4470 9376
4470 9376
4470 9376
6357 6457
6357 6457
6357 6457
6357 6457
6357 6457
6357 6457
6357 6457
6357 6457
6357 6457
6357 6457
560 9969
560 9969
560 9969
560 9969
560 9969
560 9969
560 9969
560 9969
560 9969
560 9969
3363 9338
3363 9338
3363 9338
3363 9338
3363 9338
3363 9338
3286 7501
3286 7501
3286 7501
3286 7501
3134 9825
3134 9825
1919 9792
1919 9792
1919 9792
1919 9792
1919 9792
5370 5607
5370 5607
5370 5607
5370 5607
5370 5607
5370 5607
5370 5607
5370 5607
5370 5607
5370 5607
3993 5832
2811 9751
2096 9946
100 7872
5436 7964
5436 7964
5436 7964
5436 7964
5436 7964
5436 7964
5436 7964
3088 6337
3088 6337
3088 6337
1006 9017
1006 9017
1006 9017
1006 9017
1006 9017
2916 7417
2916 7417
2916 7417
2622 5645
2622 5645
2622 5645
2000 8659
2000 8659
2000 8659
2000 8659
2000 8659
2000 8659
6229 8572
6229 8572
6229 8572
6229 8572
6545 5443
6545 5443
6545 5443
6545 5443
6545 5443
6545 5443
6545 5443
4822 8509
6682 7791
1159 8683
1159 8683
5236 6108
5236 6108
4880 7265
4880 7265
4880 7265
912 6156
912 6156
912 6156
8021 8258
8021 8258
8021 8258
8021 8258
1294 6005
2901 7512
2901 7512
2901 7512
2901 7512
2901 7512
2901 7512
2901 7512
2912 7640
2912 7640
3694 5998
3694 5998
3694 5998
3694 5998
3694 5998
8629 6280
8629 6280
8629 6280
8629 6280
8629 6280
8629 6280
8629 6280
8629 6280
3708 5201
3708 5201
3708 5201
7679 7899
7679 7899
7679 7899
7679 7899
7679 7899
4948 7713
4948 7713
4948 7713
4535 5218
4535 5218
4535 5218
4535 5218
4535 5218
4535 5218
4535 5218
4535 5218
4535 5218
1333 7919
5805 9418
213 9063
100 5588
6765 8872
6765 8872
6765 8872
6765 8872
6765 8872
6765 8872
6765 8872
6765 8872
9222 9445
7622 8704
7622 8704
7622 8704
6891 6009
6891 6009
6891 6009
7785 5036
7785 5036
7785 5036
7785 5036
7785 5036
7785 5036
745 5947
745 5947
745 5947
745 5947
1218 6439
1218 6439
7833 7424
7833 7424
7833 7424
7833 7424
7833 7424
7833 7424
7833 7424
7833 7424
7833 7424
7833 7424
7203 6952
9221 9850
9221 9850
6440 7937
6440 7937
1428 9851
1428 9851
1428 9851
1428 9851
1428 9851
1428 9851
3747 5322
3747 5322
8287 5227
8287 5227
8287 5227
8287 5227
8287 5227
8287 5227
7916 7066
7916 7066
7916 7066
7916 7066
7916 7066
7916 7066
7916 7066
7916 7066
7916 7066
7916 7066
1768 9578
1768 9578
1768 9578
1768 9578
1768 9578
1768 9578
1768 9578
1768 9578
1768 9578
8887 6239
6347 9179
6347 9179
6347 9179
6347 9179
6347 9179
6347 9179
6347 9179
6347 9179
6347 9179
6347 9179
8708 7087
8708 7087
8708 7087
8708 7087
8708 7087
8708 7087
8708 7087
8708 7087
8708 7087
8708 7087
2675 7935
7339 6675
1811 5363
6379 6208
100 8800
3367 5239
3367 5239
3367 5239
3367 5239
3367 5239
3367 5239
7682 8800
1153 3412
1153 3412
1153 3412
1153 3412
1153 3412
1153 3412
7279 4149
7279 4149
7279 4149
7060 8750
7060 8750
7060 8750
7928 3879
7928 3879
7928 3879
7928 3879
7928 3879
7928 3879
7928 3879
7928 3879
7108 53
7108 53
7108 53
237 7051
237 7051
237 7051
237 7051
1487 5593
1487 5593
1487 5593
1487 5593
1487 5593
3659 3362
3659 3362
3659 3362
3659 3362
8811 6627
8811 6627
8811 6627
8811 6627
7074 7927
7074 7927
7074 7927
7074 7927
7074 7927
7074 7927
7074 7927
7074 7927
9 3948
9 3948
9 3948
9 3948
5373 4852
7446 4852
5465 4852
7664 4852
19 1422
15 1422
16 1422
16 1422
16 7378
19 7378
14 7378
11 7378
2409 1273
7509 135
5462 8122
8184 9268
9263 2313
8230 310
8585 530
9593 2066
688 6924
593 4017
1777 5388
2949 6854
1439 9853
2676 9665
8494 4422
6082 4933
4284 1286
1111 9149
1224 4182
8792 2448
4757 861
9055 3065
2707 8585
3795 5235
9215 4818
5686 9947
6226 8002
9887 8281
4095 6251
100 6796
9945 4632
9945 4632
9945 4632
9945 4632
9945 4632
9945 4632
1078 4319
1078 4319
1078 4319
1078 4319
3077 9273
3077 9273
3077 9273
3077 9273
3077 9273
3077 9273
3077 9273
3077 9273
3077 9273
7634 1210
7634 1210
7634 1210
7634 1210
7634 1210
2918 8927
2918 8927
2918 8927
2918 8927
2918 8927
2918 8927
7083 1918
7083 1918
7083 1918
7083 1918
7083 1918
7083 1918
1814 5831
1814 5831
1814 5831
1814 5831
5787 4998
5787 4998
2531 419
2531 419
2531 419
2531 419
2531 419
2531 419
2531 419
2531 419
2531 419
2531 419
1 775
1 775
1 775
1 775
9543 6021
5038 6021
6413 6021
7565 6021
16 931
11 931
14 931
18 931
18 5865
16 5865
12 5865
13 5865
2760 8167
4349 2051
8077 1548
2623 9043
8368 487
4834 5187
1926 4255
3086 9657
3463 5857
8318 9248
7310 3742
8623 6663
3529 8978
1892 2806
7194 4825
1432 98
5867 1709
6724 4561
1604 1259
6344 5352
4546 7719
9089 4824
6393 7239
5345 6435
6927 9921
5531 523
6406 1045
6136 4744
4930 2180
9137 6582
5688 283
8935 8740
100 8697
1 15
1 15
1 15
1 15
1 15
1 15
1 15
1 15
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 11
1 11
1 11
1 11
1 11
1 11
1 11
1 11
1 13
1 13
1 13
1 13
1 13
1 17
1 17
1 17
1 17
1 17
1 17
1 8
1 8
1 8
1 8
1 8
1 8
1 8
1 8
1 5
1 5
1 5
1 5
1 5
1 5
1 5
1 13
1 13
1 13
1 18
1 18
1 18
1 18
1 18
1 18
1 18
1 11
1 11
1 11
1 11
1 11
1 11
1 11
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 18
1 18
1 18
1 18
1 18
1 18
1 18
1 11
1 2
1 16
1 12
1 5
1 8
1 16
100 5282
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 18
1 18
1 15
1 15
1 15
1 15
1 15
1 10
1 10
1 10
1 10
1 10
1 10
1 10
1 17
1 17
1 17
1 17
1 17
1 17
1 17
1 6
1 5
1 5
1 5
1 5
1 5
1 5
1 5
1 5
1 5
1 12
1 13
1 13
1 13
1 13
1 3
1 3
1 3
1 3
1 3
1 3
1 3
1 16
1 20
1 20
1 20
1 20
1 20
1 20
1 20
1 20
1 5
1 5
1 5
1 5
1 5
1 5
1 5
1 17
1 9
1 9
1 9
1 9
1 18
1 7
1 7
1 7
1 7
1 7
1 7
1 7
1 7
1 7
1 7
1 7
1 7
1 7
1 5
1 17
1 17
1 17
1 17
1 17
1 17
1 17
1 19
1 14
1 12
1 12
100 6910
1 12
1 12
1 12
1 12
1 12
1 12
1 2
1 2
1 2
1 2
1 2
1 2
1 9
1 9
1 9
1 9
1 9
1 9
1 9
1 9
1 9
1 2
1 2
1 2
1 2
1 2
1 2
1 2
1 11
1 11
1 11
1 11
1 11
1 11
1 18
1 18
1 18
1 18
1 18
1 18
1 18
1 10
1 10
1 10
1 10
1 10
1 10
1 10
1 10
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 19
1 5
1 5
1 6
1 6
1 6
1 6
1 6
1 6
1 6
1 6
1 6
1 20
1 9
1 9
1 9
1 9
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 12
1 18
1 18
1 18
1 18
1 18
1 18
1 18
1 18
1 18
1 18
1 1
