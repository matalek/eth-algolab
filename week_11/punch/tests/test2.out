5390 6
10558 8
560 1
213 1
745 1
20 2
9 4
5 5
3 3
4 4
525525 6
1047222 7
560 1
213 1
745 1
20 2
9 4
458 13
265 22
346 11
